apps/core/src/dashboards/entities/dashboard-widget.entity.ts

```diff
@@ -4,6 +4,7 @@ import { IsEnum, IsInt, IsNumber, IsObject, IsString } from 'class-validator';
  * List of available widget types.
  */
 export enum DashboardWidgetType {
```

Comment:
Suggestion
can there be a collection of widget type exported from app-kit for "book keeping" purpose? e.g. tests can ensure all widget types are tested

Reply:
I think that's the right way to go.